pig-tests/src/org/partiql/pig/tests/generated/partiql-basic.kt

```diff
@@ -1680,7 +1680,6 @@ class PartiqlBasic private constructor() {
         
             override fun hashCode(): Int = myHashCode
         }
-    
```

Comment:
nit: I'd suggest adding this newline back. Currently the generated class files look like:

```
class ProjectList(
    val items: kotlin.collections.List<ProjectItem>,
    override val metas: MetaContainer = emptyMetaContainer()
): Projection() {
    ...
    override fun hashCode(): Int = myHashCode
}
class ProjectValue(
    val value: Expr,
    override val metas: MetaContainer = emptyMetaContainer()
): Projection() {
```